apps/08jquery-towers-of-hanoi/script.js

```diff
@@ -1,5 +1,25 @@
 'use strict';
 
 $(document).ready(function() {
-  // Put app logic here
-}
+  var $block = null;
+
+  function checkForWin(){
+    if($('div[data-stack="2"]').children().length === 4 || $('div[data-stack="3"]').children().length === 4) {
+      $('div#announce-game-won').text('You Won!');
+    }
+  }
+
+  $('[data-stack]').click(function() {
+    if(!$block) {
+      $block = $(this).children().last().detach()
+    }else{
+      var $lastChild = $(this).children().last();
+      if($(this).children().length === 0 ||  $lastChild.data('block') > $block.data('block')) {
```

Comment:
Because you created the $lastChild variable, you use it here on line 17 to shorten your 'if' statement:

($lastChild.length === 0 || $lastChild.data('block') > ...) {}